spring/spring-boot/infinispan-spring-boot-starter/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.infinispan</groupId>
+        <artifactId>infinispan-spring-boot-starter-parent</artifactId>
+        <version>12.0.0-SNAPSHOT</version>
+    </parent>
+
+    <name>Infinispan Spring Boot Starter Integration Test</name>
```

Comment:
well, it says "integration test" and it doesn't contain any non-test code.

Reply:
in another PR

Reply:
why? I'm not asking to write new tests... just to move 2 classes... it is like 5 min, not 5 days of work...